tests/reftests/env.win32.test

```diff
@@ -23,14 +23,13 @@ Done.
 ### : already contains a directory which needs escaping (e.g. building with
 ### : MSVC)
 ### opam env --shell=cmd | grep "PATH" | 'XXX:.*' -> '' | 'set "P' -> 'set P'
-set "MANPATH=:"${BASEDIR}/OPAM/rewriting/man""
+set MANPATH=:${BASEDIR}/OPAM/rewriting/man
 set PATH=${BASEDIR}/OPAM/rewriting/bin;C:\Devel\bin1;C:\Devel\bin2;"C:\Devel\bin3;";C:\Devel\bin4;ZZZ:\;
 ### opam exec -- opam env --shell=cmd --revert | grep 'ZZZ:' | 'ZZZ:\\.*' -> 'ZZZ:\' | 'set "P' -> 'set P' 
 ### : Test for #5838
 ### opam env | grep MANPATH
-MANPATH=':"${BASEDIR}/OPAM/rewriting/man"'; export MANPATH;
+MANPATH=':${BASEDIR}/OPAM/rewriting/man'; export MANPATH;
 ### opam exec -- opam env --revert | grep MANPATH
-MANPATH=''; export MANPATH;
```

Comment:
is this expected?

Reply:
No, that was clumsy of me. I think the `Target` above should be `Host`, but I need to test that (especially w.r.t. MSYS2)